### ISSUE TITLE
xmlschema.validators.exceptions.XMLSchemaParseError: the QName 'abbrev-elements' is mapped to the namespace 'http://www.w3.org/1999/xhtml', but this namespace has not an xs:import statement in the schema:

### ISSUE THREAD

**Issue by WolfgangFahl** [sissaschool/xmlschema#182]
I am trying to parse thre schema from the following git repository:

    git clone --single-branch --branch Conference_ID https://gitlab.com/crossref/schema

Since i can't change the content of the schema - is there some option to make the parser more lenient and not choke on the issue?

Or is this a bug or misunderstanding of how the library should be used. I tried both:

   xmlschema.XMLSchema(path)
   xmlschema.XMLSchema11(path)

with path pointing to schema/schemas/crossref4.5.0.xsd

**Comment by brunato** [comment]
Hi,
the parsing error was related to default namespace declaration (xmlns) in this XSD file:
```
schema/schemas/Jats-journalpublishing1-mathml3-elements.xsd
```
Nothing wrong in this XSD file but with ElementTree the namespaces map is not available on parsed trees so they have to be processed with `iterparse` and this can be problematic for chameleon schemas when the default namespace is declared in some internal components (documentation in this case, that are laxly processed parts and so don't require an explicit import).

Another problem raised with this case was an UPA violation wrongly detected. The UPA checker (*distinguishable_paths* function) has been fixed and the schema can succesfully built.
The fix is available with release v1.1.3, so i close this issue (reopen if there are additional problems related to this).

Cheers

**Comment by WolfgangFahl** [comment]
thx https://github.com/WolfgangFahl/XSDToSMW/issues/1 has your version now and unit tests works
